src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -339,6 +339,8 @@ injectRoute({
 });
 ```
 
+When publishing the above mentioned example package `@fancy/dashboard` to npm, it's required to export the `dashboard.astro` file in the `package.json` export section as follows: `"exports": {"./dashboard.astro": "./dashboard.astro"}`. Otherwise, the file will be looked up in the project folder, which will cause a `no such file or directory` exception.
```

Comment:
``````suggestion
When publishing your package (`@fancy/dashboard`, in this case) to npm, you must export `dashboard.astro` in your `package.json`. Otherwise, the file will be looked up in the project folder, which will cause a `no such file or directory` exception.

```json {"exports"}
{
  "name": "@fancy/dashboard",
  // ...
  "exports": { "./dashboard.astro": "./dashboard.astro" }`
}
```
``````